src/Modal/Modal.component.tsx

```diff
@@ -30,32 +28,18 @@ export type Props = ReactModal.Props & {
    **/
   modalTitle: string;
   /**
-   * Specify the function to execute when the user clicks Cancel or No.
```

Comment:
For the `footerContent` prop, With propType defined as `any`, am able to pass `strings, arrays, etc..`. The component does not throw any error or warning, but does not show this content either.

I think using the following would be more appropriate:
`footerContent: JSX.Element | React.Component | React.FunctionComponent | string;`

This way, the correct warnings are provided if incorrect props are passed. Also, it would be possible to display the footer using single line ` {this.props.footerContent}` instead of `{this.renderFooter(this.props.footerContent)}`